server/src/main/resources/templates/web/layouts/main.html

```diff
@@ -8,7 +8,10 @@
                 companyName=${branding.getZoneCompanyName() ?: ''},
                 copyrightCompanyName=${branding.getCompanyName() ?: 'General Electric Company'},
                 copyrightFallback=${'Copyright &#169; ' + #dates.year(#dates.createNow()) + ' ' + copyrightCompanyName + '. All Rights Reserved.'},
-                copyright=${branding.getFooterLegalText() ?: copyrightFallback}" >
+                copyright=${branding.getFooterLegalText() ?: copyrightFallback},
+                brandingLogo=${branding.getProductLogo()},
+                fallbackLogo=${assetBaseUrl+'/images/predix-word.svg'},
+                logo=${brandingLogo != null ? 'data:image/png;base64,'+#strings.replace(brandingLogo,T(java.lang.System).getProperty('line.separator'),'') : fallbackLogo}" >
```

Comment:
I assume that main login page wasn't showing branding before then?

Reply:
This can be removed too as it is all handled on line 29.